public/stylesheets/site/2.0/21-userstuff.css

```diff
@@ -1,10 +1,11 @@
 /* ==USERSTUFF: displaying content inputted by a user into a textarea.*/
 
 .userstuff {
-    word-wrap: break-word;
+  word-wrap: break-word;
```

Comment:
Could you put the spacing back here? Our house style (developed many, many years ago) is to list CSS3 properties last and indent them four spaces.

Reply:
Ah, yeah. That's my linter. Let me turn that off for this.

Reply:
It looks like this line got fixed! However, all of the other properties are now indented 2 spaces instead of 4. Can you please restore the original indenting across the file?

Reply:
All lines should have been fixed with that commit. I just checked again on the diff and it all looks good? 
![Screenshot from this morning, from my IDE too.](https://github.com/otwcode/otwarchive/assets/127066227/f9774fa0-53eb-4579-b92e-12d055879f4f) Not sure if GH is just being slow to catch up, or what.

Reply:
The screenshot matches what GitHub is showing, but the properties should be indented just two spaces, putting them where the second vertical line is. Basically, the lines were correctly indented in the original version of the file and should be unchanged in this PR.

Reply:
Wait, so then I'm confused about the original comment, asking to indent them all 4 spaces?

Or does that mean just for CSS3 properties (indent 4), and previous versions are all indented 2?

Reply:
Right, just the CSS3 properties get indented 4 spaces (and go last), and the rest get indented 2. Sorry for the confusion!